shared/common-adapters/avatar.render.desktop.js

```diff
@@ -56,19 +56,13 @@ const BackgroundDiv = glamorous.div(
     top: backgroundOffset,
   },
   props => ({
-    backgroundColor: props.loaded ? globalColors.white : props.loadingColor || globalColors.lightGrey,
```

Comment:
turns out the loading flow got broken awhile ago. what happens is it starts out with the placeholder avatar, it tries to load it and it sees its loaded and so loaded is basically always true. so we can rip this all out and it looks 100% the same